test/Performance/setup_env_unix.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+set -e
+if [[ ("$1" = "Ubuntu16" || "$1" = "RedHat7" || "$1" = "Sierra") ]]; then
+    PLATFORM=$1
+else
+    echo "First argument must be one of Ubuntu16, RedHat7, Sierra. Exiting..."
+    exit 1
+fi
+if [[ "$2" = "" ]]; then
```

Comment:
Changed to `if [[ "$2" != 7.*.* ]]; then`
For Unix, php download page is the same for both latest and older versions of PHP.
For Windows, the latest source is in http://windows.php.net/downloads/releases/,
the older versions are in http://windows.php.net/downloads/releases/archives/